apps/portals/adknowledgeportal/src/config/routesConfig.tsx

```diff
@@ -40,6 +42,7 @@ const routes: RouteObject[] = [
     ),
     children: [
       ...sharedRoutes,
+      getDoiRedirectRoute(doiRedirectConfig),
```

Comment:
I see, so we would have a new route for every resource type (and a new doiRedirectConfig)?

Reply:
What if a single portal wants to support DOIs for more than one resource, say Study and Dataset?

Reply:
the resource is encoded into the ID. So the backend will send a request to

`https://theportal.synapse.org/doi?id=STUDY.syn123`
`https://theportal.synapse.org/doi?id=DATASET.2d4c899`

(we discussed elsewhere that this format of IDs is probably not appropriate, but it illustrates the idea)